components/arm/arm_test.go

```diff
@@ -60,97 +60,163 @@ func TestStatusValid(t *testing.T) {
 }
 
 func TestCreateStatus(t *testing.T) {
```

Comment:
I wanted to add tests for my changes to `CreateStatus` and had to refactor the existing tests in order to be able to reuse shared variables/functions. The changes ensure that the tests don't overwrite the shared variables/functions, which decouples them from each other and makes them more robust.